### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Table_Table Grid_Name Role Value: No Name is defined for sorting button.

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/AzureStorageExplorer#7610]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1(96)

### Regression From

NO

### Architecture

x64

### Storage Explorer Build Number

20231114.10

### Platform

macOS

### OS Version

MAC Version 13.6.1

### GitHub Tags:
#A11yMAS;#A11yTCS;#MAC;#DesktopApp;#A11ySev2;#BM_AzureToolsStorageExplorer_MAC_Dec2023;#Azure Tools Storage Explorer;#FTP;#Voiceover;#WCAG4.1.2;#Name Role Value;#AILimited;

### Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.32.1 (96)
Screen Reader: Voice Over

### Steps to Reproduce

1. Launch Storage Explorer
2. Tab Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Tab Navigate to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Tab Navigate to "Tables" of the selected subscription and expand it.
5. Tab Navigate to table.
6. Turn on Voice Over and Observe that No Name is defined for sorting button.

### Actual Experience

No Name is defined for sorting button.

### Expected Experience

Name should be defined for sorting button.

### User Impact:
Screen reader users will face difficulty if No Name is defined for sorting button.

### Attachment:
![A11y_Azure Tools Storage Explorer_Table_Table Grid_Name Role Value](https://github.com/microsoft/AzureStorageExplorer/assets/98728920/057c4423-2557-4038-a422-53a680993615)

**Comment by craxal** [comment]
@keerthiduvvuri Can you provide additional details? There is no actual button there, and you shouldn't be able to focus the sort "button", because it's not a focusable element. Only the grid itself can be focused, and the arrow keys change which cell has focus.

**Comment by keerthiduvvuri** [comment]
@Craxal in that grid there is a sorting functionality present and Keyboard focus is going to that because it is interactive and when we navigate with Voice over it is just announcing as image, Image is not a name. the name property should be defined as "sort by or sorting data".

**Comment by craxal** [comment]
@keerthiduvvuri I'm still not clear on a number of points:
* How are you navigating to the image? That is not normally accessible with tab and arrow keys. Without knowing this, I am not able to reproduce this issue.
* The image itself is not an interactive element. The header cell should be interactive. It's not clear as to how you are interacting with the sorting mechanism. (Also, "Sorting button" is not a valid role, but "button" is a valid role.)
* This may be more of a question of your testing procedure, but why are you turning VoiceOver on after navigating to the table? Why wouldn't you have VoiceOver turned on from the beginning? Does that produce different results?

**Comment by kupatkar99** [comment]
Here expected is that the name property as "Sort ascending or sort descending" should be provide to the "Arrow" control present in the column header as when we are pressing "SPACE" key on it the sort functionality is performing.

**Comment by craxal** [comment]
@kupatkar99 The button role should belong to the entire header content, not the image, because clicking anywhere in the header will sort the column.

**Comment by kupatkar99** [comment]
verified the issue in below environment:
Microsoft Azure Storage explorer
Version:
As per the above comment the button role should belong to the entire header content, not the image, because clicking anywhere in the header will sort the column. Parent name property is defined which belongs to the complete coloumn header. hence closing the bug. please find below added attachment for refernce
<img width="481" alt="image" src="https://github.com/microsoft/AzureStorageExplorer/assets/98530069/cf14f222-d40a-466b-aaf1-4bacd520ec50">
